### PR TITLE
add configuration sample for SimpleProvider

### PR DESCRIPTION
**Pull request by krichprollsch**

### CODE REVIEW

File: Resources/doc/configuration.md [hunk 1]
```diff
@@ -110,6 +110,28 @@ services:
...
+A provider to add static routes into the sitemap easily.
+
+```yml
+parameters:
+    simple_providers_options:
+        routes:
+            - {name: homepage}
+            - {name: foo, params: {foo: bar}, lastmod: '2015-01-01', changefreq: monthly, priority: 0.5}
+    # the following parameters are optionnal
+    lastmod:        '2015-01-01'
```
Comment (K-Phoen) [inline Resources/doc/configuration.md]: Isn't there an indentation problem here?

**Comment by krichprollsch** [comment]
fixed but not yet tested.
I could test tomorrow

**Comment by K-Phoen** [comment]
Works for me.

I'm not really happy with the way we currently define providers... having to define parameters and services is a bit weird. I'll try to clean that soon (probably next week)

**Comment by krichprollsch** [comment]
Well, I think the config is very easy to define custom providers... I like combo services + options + tags way.
But maybe it's harder for newcomers.

**Comment by K-Phoen** [comment]
I also like the combo service + tag. But as you said, it could be harder for newcomers. Especially since there is no real validation on the options given to the services.

I'd like to provide an alternative way to create providers (probably configuration-based). I'll come up with a draft soon.

**Pull request closed without merging**